Formula/vsh.rb

```diff
@@ -0,0 +1,19 @@
+class Vsh < Formula
+  desc "HashiCorp Vault interactive shell"
+  homepage "https://github.com/fishi0x01/vsh"
+  url "https://github.com/fishi0x01/vsh/archive/v0.7.2.tar.gz"
+  sha256 "f20885425a9c6ee04ed447046e6df113585469b37332e02b7a23639b718314a1"
+  license "MIT"
+
+  depends_on "go" => [:build]
+
+  def install
+    system "go", "build", "-ldflags", "-X main.vshVersion=v#{version}", "-o", "build/vsh"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "-ldflags", "-X main.vshVersion=v#{version}"
```
I _think_ this might also make the `bin.install` line unnecessary.

Reply:
Good to know. Added it and removed `bin.install` :+1: